src/Parser/PersonParser.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Defr\Parser;
+
+use Defr\Parser\Helper\StringHelper;
+use Defr\ValueObject\Person;
+use Symfony\Component\DomCrawler\Crawler;
+
+final class PersonParser
+{
+    /**
+     * @param Crawler $crawler
+     *
+     * @return Person
+     */
+    public static function parseFromDomCrawler(Crawler $crawler)
+    {
+
+        $person = $crawler->filter('.div-cell')->each(function (Crawler $node, $i) {
+            $content = StringHelper::removeEmptyLines($node->text());
+
+            $contentItems = explode("\n", $content);
+            $contentItems = array_map('trim', $contentItems);
+
+            if ($i === 0) {
+                if (mb_strpos($contentItems[0], 'ednatel') !== false) {
+                    return ['type' => 'executive'];
+                }
+                return ['type' => 'partner'];
+            }
+
+            if (count($contentItems)) {
+                if (mb_strpos($contentItems[0], ', dat. nar.') !== false) {
+
+                    $name = trim(explode(', dat. nar.', $contentItems[0])[0]);
+
+                    try {
+                        $birthday = DateTimeParser::parseFromCzechDateString($contentItems[1]);
+                        $address = $contentItems[2];
+                    } catch (\Exception $e) {
+                        $birthday = null;
+                    }
+
+                    return ['name' => ucwords(mb_strtolower($name)), 'birthday' => $birthday, 'address' => isset($address) ? $address : ''];
```

Comment:
I have found that `ucwords` does not handle words starting with diacritics properly. Let me suggest a fix I did in https://github.com/filipmelik/Ares/commit/c692788fa98cd94af4d5664f08fc3e04de8f121f (along with some code style changes)